### DISCUSSION TITLE
Should @angular dependencies be "devDependencies"?

### DISCUSSION THREAD

**Discussion by TheSpiciestDev** [nrwl/nx#7052] [General]
Because my built NestJS projects use my package.json's production dependencies for their runtimes, I usually move all of my `@angular/*` packages to the `devDependencies` object because I do not want Angular to be bundled up with my NestJS artifacts. Does anyone else move around their frontend dependencies to avoid the backend's including them?.. should this be standard?

**Comment by ryancraigmartin** [comment]
Hey @METACEO! Did you have to start moving the deps to devDeps because you were seeing angular dependencies in your generated package.json when building the API? If so, I would reccomend running `nx dep-graph` and see what your API is linked to. You might have a dependency issue like an accidental Angular module import where a Nest module import should be. I had run into this issue before. Hope it helps lead you in the right direction.
